alpha_1: 0.1  # meters/meter
alpha_2: 0.1 # meters/degree
alpha_3: 0.1   # degrees/meter
alpha_4: 0.1  # degrees/degree

# ICP PARAMETERS
max_correspondence_distance: 0.1 
max_iterations: 2000
ransac_iterations: 2000
ransac_outlier_threshold: 0.1
icp_score_scale: 600.0
icp_optimization_epsilon: 0.0000001

initial_pose_x: 0.0
initial_pose_y: 0.0
initial_pose_a: 0.0

initial_cov_xx: 0.05
initial_cov_yy: 0.05
initial_cov_aa: 0.05

update_min_d: 0.001
update_min_a: 0.001

use_map_topic: false
first_map_only: true

rate: 50
covariance_marker_scale: 2
